avatars/api.py

```diff
@@ -233,14 +222,31 @@ def create_job(
         *,
         timeout: Optional[int] = DEFAULT_TIMEOUT,
     ) -> Job:
-        """Create a job."""
+        """Create a job.
+
+        Deprecated. Use create_avatarization_job instead.
+        """
         kwargs = {
             "method": "post",
             "url": f"/jobs",
         }
 
         return Job(**self.client.request(**kwargs, json=request, timeout=timeout))
```

Comment:
What about calling `create_avatarization_job` while changing the parameters as users should do according to us ?

Reply:
If I understand you correctly, you would modify `create_job` to call `self.jobs.create_avatarization_job` ?

That would work, except it would be ugly code in the jinja templates, as we would need to isolate the `create_job` operation and apply special logic to it, compared to just letting it be autogenerated now.
And that wouldn't change how the R client behaves, nor the demo. 

There is no free lunch, and I think the option that I chose is the one that is the easiest to maintain.